pages/index.tsx

```diff
@@ -22,13 +22,25 @@ import { fromTxError } from "@/utils/errors"
 export default function Home() {
   const wallet = useWallet()
   const { publicKey } = wallet
-  const { connection } = useConnection()
+  const connection = new Connection(process.env.NEXT_PUBLIC_RPC as string)
```

Comment:
Should use the connection from the `useConnection` hook because it's already initialized